action.md

```diff
@@ -27,4 +31,36 @@ jobs:
           OPENAI_API_KEY: ${{ secrets.OPENAI_API_KEY }}
           MODEL: 'gpt-4o'
           GITHUB_TOKEN: ${{ github.token }}
-```
\ No newline at end of file
+```
+
+### Workflow yml option 2: Add a code review bot
+
+In this config, a GPT CR is triggered when a specific user is added as a "reviewer" in the Github UI. Create an additional Github account such as 'YourProject-ML-CR-bot', then speficy the account username in the config.
+
+This option can save on API costs by only reviewing when explicity asked to. It can also be used to avoid reviewing PRs before they are ready (draft/WIP PRs).
+
+To trigger a re-review, simply remove and re-add the bot to the reviewers list.
+
+```yaml
+name: Code Review GPT
+
+on:
+  pull_request:
+    types: [ review_requested ]
+
+jobs:
+  run_code_review:
+    if: ${{ github.event.requested_reviewer.login == 'YourProject-ML-CR-bot'}}
```

Comment:
unsure if this would actually work unless you deployed as an app.

Reply:
it works, I was running it. It has to be "on: pull_request: review_requested" event, but that filter is above.

Reply:
how would it work if it's not deployed? what would `YourProject-ML-CR-bot` be..

Reply:
Not sure I follow. 

“YourProject-ML-CR-bot” is a GitHub user ID. One you create. 

The yaml is already live. 

This is just to manually control when CR is run, as compared to every commit on every PR.